demo/web/firebase_element/app_element.html

```diff
@@ -0,0 +1,39 @@
+<dom-module id="app-element">
+  <template>
+    <style>
+      :host {
+        display: block;
+      }
+    </style>
+
+    <div class="horizontal center-justified layout">
+      <div class="vertical-section">
+        <h3>Dinosaurs by Height</h3>
+        <template is="dom-bind">
+          <firebase-collection
+              order-by-child="height"
+              location="https://dinosaur-facts.firebaseio.com/dinosaurs"
+              data="{{dinosaurs}}"></firebase-collection>
+          <template is="dom-repeat" items="[[dinosaurs]]" as="dinosaur">
```

Comment:
So if we modify `polymer_elements/lib/src/paper-styles/demo-pages.html` it will just get bashed over the next time we update, which I would like to avoid. You could for now just have a custom `demo-pages.html` file in the `demo` project though. Really this change needs to be made upstream, but as it would presumably break a number of JS demos that might be an issue.